lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1683,15 +1684,14 @@ def jsonrpc_claim_abandon(self, txid, nout):
             txid : txid of resulting transaction if succesful
             fee : fee paid for the transaction if succesful
         """
-        def _disp(x):
-            log.info("Abandoned name claim tx " + str(x))
-            return self._render_response(x)
 
-        d = defer.Deferred()
-        d.addCallback(lambda _: self.session.wallet.abandon_claim(txid, nout))
-        d.addCallback(_disp)
-        d.callback(None)  # TODO: is this line necessary???
-        return d
+        try:
+            abandon_claim_tx = yield self.session.wallet.abandon_claim(txid, nout)
+            response = yield self._render_response(abandon_claim_tx)
+        except Exception as err:
+            log.warning(err)
```

Comment:
use `exc_info` in log statements that you want to include the error message.

This should be more like, `log.warning('Failed to abandon claim %s - %s', txid, nout, exc_info=True)`